.github/workflows/nighly_fuzzing.yml

```diff
@@ -19,28 +19,38 @@ jobs:
         with:
           enable-cache: true
           cache-dependency-glob: uv.lock
+
       - uses: actions/setup-python@v5
         with:
           python-version-file: ./cairo/.python-version
+
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
         with:
           version: nightly
+
+      - name: Install forge deps
+        run: forge install
+
+      - name: Build solidity contracts
+        run: forge build
+
       - name: Run tests
         id: tests
         continue-on-error: true # To send a notification if the tests fail
         run: |
           echo "HYPOTHESIS_PROFILE=nightly" >> .env
           cd cairo
           uv run compile
-          forge build
-          uv run pytest "tests" -m "not NoCI" -n logical --junitxml=junit.xml -o junit_family=legacy
+          uv run pytest "tests" -m "not NoCI" -n logical --junitxml=junit.xml -o junit_family=legacy --durations=0
```

Comment:
```suggestion
          uv run pytest "tests" -m "not NoCI" -n logical --junitxml=junit.xml -o junit_family=legacy --durations=0 -s --hypothesis-verbosity=debug
```

Reply:
We cannot use this due to too much log generated. 
See comment https://github.com/kkrt-labs/keth/issues/259#issuecomment-2539149408 and related PR https://github.com/kkrt-labs/keth/pull/262#issue-2736183559 and